composer.json

```diff
@@ -46,11 +46,11 @@
     "require-dev": {
         "ext-json": "*",
         "friendsofphp/php-cs-fixer": "^3.14",
-        "laravel-json-api/laravel": "^2.0|^3.0|^4.0",
+        "laravel-json-api/laravel": "^2.0|^3.0|^4.0|^5.0",
         "laravel-json-api/non-eloquent": "^2.0|^3.0|^v4.0",
         "nesbot/carbon": "^2.63|^3.0",
         "orchestra/testbench": "^6.25|^7.21|^8.0|^9.0",
-        "phpunit/phpunit": "^9.5"
+        "phpunit/phpunit": "^9.5|^10.0|^11.0"
```

Comment:
I noticed that you're requiring multiple versions of phpunit. There are some compatibility issues with the current configuration file. Let's reduce it to `^10.0` for now and lets set the minimum PHP version to `8.1` as `8.0` has been [end of life](https://www.php.net/supported-versions.php) for almost a year now.